### PR TITLE
ref(sentry10): Remove Sentry 9 releases code

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: src/sentry/static/sentry/app/routes.jsx [hunk 1]
```diff
@@ -1183,58 +1183,6 @@ function routes() {
             }
             component={errorHandler(LazyLoad)}
           />
-          <Route
-            path="releases/"
```
Comment (billyvg) [inline src/sentry/static/sentry/app/routes.jsx]: Can we redirect these old URLs?

Comment (lynnagara) [reply src/sentry/static/sentry/app/routes.jsx]: We could. I'd personally vote to simplify our codebase and drop support though. Apart from a handful of opt outs, I think people shouldn't have been using these routes for months now, and I don't think we should commit to supporting these forever.

Comment (markstory) [reply src/sentry/static/sentry/app/routes.jsx]: Users could have bookmarks.

Comment (lynnagara) [reply src/sentry/static/sentry/app/routes.jsx]: For any route except list where it's nested under a specific version it seems unlikely due to our retention policy

**Review by billyvg — APPROVED** [review]
🎉

**Pull request merged**